django/applications/catmaid/control/cropping.py

```diff
@@ -9,7 +12,13 @@
 from catmaid.models import Stack, Project, ProjectStack, Message, User
 from catmaid.control.common import id_generator, json_error_response
 
-import urllib2 as urllib
+try:
```

Comment:
Might want to comment this block here so that it's clear `urllib` is for 3 and `urllib2` is for 2.